include/zephyr/sys/util.h

```diff
@@ -517,7 +628,7 @@ char *utf8_lcpy(char *dst, const char *src, size_t n);
 #define GB(x) (MB(x) << 10)
 
 /** @brief Number of Hz in @p x kHz */
-#define KHZ(x) ((x) * 1000)
+#define KHZ(x) ((x)*1000)
```

Comment:
This looks wrong, since it doesn't match the spacing in the `MHZ()` macro below.

Reply:
That's just auto-formatted according to Zephyr's `.clang-format`, but I can revert it since it's unrelated.